### PR TITLE
[Hotfix] 2.6.8

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

File: api/views/workflow.py [hunk 1]
```diff
@@ -35,8 +36,8 @@ class SourceViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = SourceSerializer
 
     def get_queryset(self):
-        queryset = Source.objects.exclude(icon='')
-        sort = self.request.query_params.get("sort")
+        queryset = Source.objects.exclude(models.Q(icon='') | models.Q(is_active=False))
```
Comment (chrisseto) [inline api/views/workflow.py]: I'll add tests for graphql.
Sometimes I hate boolean logic, 👍  will change.

**Comment by chrisseto** [comment]
@leb2dg is_active will stop them from showing up on the sources page and the sources ES index. Should I go ahead and exclude them from the sources field on creative works?

**Comment by leb2dg** [comment]
As a user, if there is data from a source, I would expect the source to be listed/shown. If there is no data for a source, I would not expect the source to be shown/listed.

I guess my question is what use cases is the `is_active` flag covering?

**Comment by chrisseto** [comment]
@leb2dg `is_active` is being added to avoid having to delete Biorxiv RSS. Does that answer your question?

File: share/management/commands/loadsources.py [hunk 2]
```diff
@@ -23,8 +21,14 @@ def add_arguments(self, parser):
         parser.add_argument('--overwrite', action='store_true', help='Overwrite existing sources and source configs')
 
     def handle(self, *args, **options):
+        global apps
```
Comment (aaxelb) [inline share/management/commands/loadsources.py]: Eeeeeeew. But fair enough, it has to work somehow...

File: bots/elasticsearch/tasks.py [hunk 3]
```diff
@@ -120,7 +120,7 @@ def bulk_stream(self, es_index):
 
         for source in Source.objects.all():
             # remove sources from search that don't appear on the sources page
-            if not source.icon:
+            if not source.icon or not source.is_active:
```
Comment (aaxelb) [inline bots/elasticsearch/tasks.py]: `is_deleted`

File: share/management/commands/loadsources.py (hunk 2 above)
Comment (chrisseto) [inline share/management/commands/loadsources.py]: It's probably a better idea to just tack this onto `self`. Not sure why I didn't think of that first.

Comment (aaxelb) [inline share/management/commands/loadsources.py]: Yeah, that is better.

**Pull request merged**